package.json

```diff
@@ -186,11 +186,11 @@
 	},
 	"lint-staged": {
 		"*.scss": [
-			"npm run lint:css"
+			"npm run lint:css -s"
 		],
 		"*.js": [
 			"prettier --write",
-			"npm run lint:js"
+			"npm run lint:js -s"
```

Comment:
🤔 I don't think I've seen the `-s` flag before (nor see it defined [in the cli args list](https://eslint.org/docs/2.13.1/user-guide/command-line-interface))

Reply:
Odd it worked... silent mode for `npm` to avoid all the error text added after the lint.

Reply:
ahhh, so an npm flag (not eslint) 👍